### ISSUE TITLE
Update quantecon.syzygy.ca to use Julia 1.2 kernel

### ISSUE THREAD

**Issue by jlperla** [QuantEcon/lecture-source-jl#646]
Right now if you launch a notebook, it asks you to select a kernel (because they are set to Julia 1.2 instead of the 1.1 that is installed).

- [x] VSE 
- [ ] QuantEcon

**Comment by arnavs** [comment]
This is also dependent on the upstream jupyter/datascience-notebook. Once that’s done it’s just a find and replace.

**Comment by jlperla** [comment]
I made a comment on the PR, but I am concerned that this is not the right approach.  

Maybe the best approach is to take the dockerfile and do a QE specific one where we have better control and can cut out what we don't need in order to speed things up.  Image sizes matter, and I suspect there is a whole bunch of stuff this image has that we don't need.  They can't (and shouldn't) accept our changes because they may be too specific.

Basically, create a docker on qe dockerhub called `quantecon-julia-notebook` where:
- To begin with (i.e. can we do it immediately?) it simply has https://github.com/jupyter/docker-stacks/blob/24f10a5c39ba5295617709cd38458d97ef26a413/datascience-notebook/Dockerfile directly
- Then we plan to eventually go back through the image dependencies to try to minimize all of the crap that it installs.

Does that make sense?  If so, is there a way to initiate the dockerhub build with it and then we could just swap out `scipy-notebook` with `quantecon-julia-notebook` in our syzygy and binder builds?

**Comment by jlperla** [comment]
For example, wouldn't dependence on this mean we have the full R stack in the binder kernels due to using the `datascience-notebook`?

**Comment by arnavs** [comment]
OK, I updated the kernel for VSE syzygy by making our notebook by changing our dependence to `jupyter/scipy-notebook` and then including stuff for Julia etc. It's building now on the server. @jlperla can you play around with this on Sunday and let me know if things look OK for teaching?

For the QE, I will use our sageless minimal Dockerfile from before.

**Comment by jlperla** [comment]
Sure, just tell me when the image is uploaded for the VSE syzygy (and if I need to delete my `.julia`, etc.).  At this point, only the 1.1 kernel is showing.

**Comment by arnavs** [comment]
@jlperla I started a build of the `quantecon.syzygy.ca`, will restart the hub when it's done and then take a look. This is the minimal approach we discussed before. It's midnight here so won't be able to verify until tomorrow. 

If for any reason it doesn't work, as a band-aid we can take the VSE syzygy, and delete the install steps corresponding to [SageMath](https://github.com/ubcecon/vse-jupyterhub/blob/master/Dockerfile#L84-L88) and [R](https://github.com/ubcecon/vse-jupyterhub/blob/master/Dockerfile#L40-L64). So then it would just be Python and Julia.
